concordium-consensus/src/Concordium/Scheduler/ProtocolLevelTokens/Module.hs

```diff
@@ -171,3 +171,19 @@ executeTokenHolderTransaction sender tokenParam = do
     tokenParamLBS =
         BS.Builder.toLazyByteString $ BS.Builder.shortByteString $ parameterBytes tokenParam
     failTH = pltError . encodeTokenHolderFailure
+
+-- | Execute a token-holder transaction. The process is as follows:
+--
+--   - Decode the transaction CBOR parameter.
+--   - Check that amounts are within the representable range.
+--   - For each transfer operation:
+--
+--       - Check that the recipient is valid.
+--       - Transfer the amount from the sender to the recipient, if the sender's balance is
+--         sufficient.
+executeTokenGovernanceTransaction ::
+    -- (PLTKernelUpdate m, PLTKernelFail EncodedTokenRejectReason m, Monad m) =>
+    PLTAccount m ->
+    TokenParameter ->
+    m ()
+executeTokenGovernanceTransaction _sender _tokenParam = error "Not implement yet. This should be implemented as part of https://linear.app/concordium/issue/COR-687"
```

Comment:
```suggestion
executeTokenGovernanceTransaction _sender _tokenParam = error "Not implemented yet. This should be implemented as part of https://linear.app/concordium/issue/COR-687"
```

Reply:
Hope you don't mind too much, but I'll leave it in to avoid waiting another 40 minutes on the CI :sweat_smile: 
This will be replaced anyway, probably during this week.

Reply:
I don't. Just noticed it 😄